comskip.c

```diff
@@ -700,7 +700,7 @@ unsigned int		min_black_frames_for_break = 1;
 bool				detectBlackFrames;
 bool				detectSceneChanges;
 int             dummy1;
-unsigned char*		frame_ptr;
+unsigned char*		frame_ptr = 0;

```

Comment:
I saw a similar bug recently in 170951cd6860d941c02125e59c11298ad8e1c3b6, but that one also only manifested sometimes on particular platforms. Again, I don't understand why it wouldn't be initialized to 0 as stated in the C specification.